index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:wght@400;500&family=Poppins:wght@400;500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Personal Portfolio Website</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <div id="header">
+                <div class="mobile-header">
+                     <div id="primary-logo">
+                         <div id="mobile-logo">
+                         </div>
+                         <div id="logo"></div>
+                     </div>
+                     <div id="mobile-menu">
+                       
+                     </div>
+                 </div>
+        </nav>
+        <nav>
+                 <div class="main-header">
+                     <div class="main-menu">
+                         <ul>
+                             <li><a href="#"> Home</a></li>
+                             <li><a href="#">Portfolio</a></li>
+                             <li><a href="#">About</a></li>
+                             <li><a href="#">Contact</a></li>
+                         </ul>
+                     </div>
+                 </div>
+                </div>
+             </div>
+        </nav>
+    </header>
+    <section>
+        <div class="headline">ggested
+            <div>
```

Comment:
![image](https://user-images.githubusercontent.com/78251927/235544648-32cd1ae6-5bca-45a2-a832-c3991c0a1fdf.png)
 
Your page has an additional text which is likely to be a typo, 

Please kindly remove this text so that your website will look exactly like the design (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).